x-pack/plugins/lens/readme.md

```diff
@@ -12,3 +12,31 @@ Run all tests from the `x-pack` root directory
 - API Functional tests:
   - Run `node scripts/functional_tests_server`
   - Run `node ../scripts/functional_test_runner.js --config ./test/api_integration/config.ts --grep=Lens`
+
+
+## UI Terminology
+
+Lens has a lot of UI elements - to make it easier to refer to them in issues or bugs, this is a hopefully complete list:
+
+* **Top nav** Navigation menu on top of the app (contains Save button)
+  * **Query bar** Input to enter KQL or Lucene query below the top nav
+  * **Filter bar** Row of filter pills below the query bar
+  * **Time picker** Global time range configurator right to the query bar
+* **Data panel** Panel to the left showing the field list
+  * **Field list** List of fields separated by available and empty fields in the data panel
+  * **Index pattern chooser** Select element switching between index patterns
```

Comment:
at some point we'd probably like to change this part to not specify that datasource is index pattern but I am good with leaving it till we don't introduce second datasource.